tox.ini

```diff
@@ -36,15 +36,17 @@ envdir = .tox/docs
 deps =
   -r requirements-dev.txt
   sphinx-intl
-commands = sphinx-build -b html {posargs} {toxinidir}/docs {toxinidir}/docs/_build/html
+  qiskit-aer
+commands = sphinx-build -W -b html {posargs} {toxinidir}/docs {toxinidir}/docs/_build/html
```

Comment:
The `W` flag treats warnings as failures when building the docs. Since the sphinx warnings are fixed with this PR, then setting the flag is likely ideal.